arch/arm/src/imxrt/imxrt_flexcan.c

```diff
@@ -1183,6 +1198,27 @@ static uint32_t imxrt_waitmcr_change(uint32_t base, uint32_t mask,
   return false;
 }
 
+static uint32_t imxrt_waitesr2_change(uint32_t base, uint32_t mask,
+                                       uint32_t target_state)
```

Comment:
Hi @michallenc, since the "state" is bool shouldn't it be better if target_state also be bool ?